### PR TITLE
chore(docs): Add openshift-ci debug documentation

### PR DESCRIPTION
**Pull request by SimonBaeumer**
### Description

Add documentation to access the running cluster and how to find build logs.

### CODE REVIEW

File: .openshift-ci/README.md [hunk 1]
```diff
@@ -28,3 +28,25 @@ make style
 make fix-style
 make lint
 ```
+
+## Debugging
+
+### Links
+
+Job Definition: https://github.com/openshift/release/tree/master/ci-operator/jobs/stackrox/acs-fleet-manager
+Config Definition: https://github.com/openshift/release/tree/master/ci-operator/config/stackrox/acs-fleet-manager
```
Comment (tommartensen) [inline .openshift-ci/README.md:37]: Shouldn't this point to `stackrox/stackrox` jobs and config?

```suggestion
Job Definition: https://github.com/openshift/release/tree/master/ci-operator/jobs/stackrox/stackrox
Config Definition: https://github.com/openshift/release/tree/master/ci-operator/config/stackrox/stackrox
```

File: .openshift-ci/README.md [hunk 2]
```diff
@@ -28,3 +28,25 @@ make style
...
+
+#### Access Job Cluster and Real Time Logs
+
+- To access the OpenShift UI to view the logs directly search for something like `Using namespace hUsing namespace https://console.build01.ci.openshift.org/k8s/cluster/projects/ci-op-yz5q9nlt`.
```
Comment (tommartensen) [inline .openshift-ci/README.md:41]: which logs and search where?

File: .openshift-ci/README.md [hunk 3]
```diff
@@ -28,3 +28,25 @@ make style
...
+- Access OpenShift UI, open `Administartor` overview on the top left.
```
Comment (tommartensen) [inline .openshift-ci/README.md:42]: [typo]

```suggestion
- Access OpenShift UI, open `Administrator` overview on the top left.
```

File: .openshift-ci/README.md [hunk 4]
```diff
@@ -28,3 +28,25 @@ make style
...
+- View the `Environment`, copy the `KUBECONFIG` path, open the Pod's `Terminal` view in the UI and run `cat <KUBECONFIG_PATH>`
+- Copy KUBECONFIG content and create the KUBECONFIG locally
+- Run `export KUBECONFIG=/local/path/to/kubeconfig`
+
+#### Check Fleet-Manager logs and build logs
```
Comment (tommartensen) [inline .openshift-ci/README.md:47]: also stackrox logs here?

File: .openshift-ci/README.md [hunk 5]
```diff
@@ -28,3 +28,25 @@ make style
...
+
+Path in articafts:
```
Comment (tommartensen) [inline .openshift-ci/README.md:49]: [typo]

```suggestion
Path in artifacts:
```

**Review by tommartensen — CHANGES_REQUESTED** [review]

File: .openshift-ci/README.md (hunk 4 above)
Comment (SimonBaeumer) [reply .openshift-ci/README.md]: done

File: .openshift-ci/README.md (hunk 5 above)
Comment (SimonBaeumer) [reply .openshift-ci/README.md]: done

File: .openshift-ci/README.md (hunk 3 above)
Comment (SimonBaeumer) [reply .openshift-ci/README.md]: done

File: .openshift-ci/README.md (hunk 1 above)
Comment (SimonBaeumer) [reply .openshift-ci/README.md]: done

File: .openshift-ci/README.md (hunk 2 above)
Comment (SimonBaeumer) [reply .openshift-ci/README.md]: done

**Review by tommartensen — APPROVED** [review]

**Pull request merged**